wizard/WizardAskPassword.qml

```diff
@@ -83,7 +83,7 @@ ColumnLayout {
 
     WizardHeader{
         title: qsTr("Give your wallet a password") + translationManager.emptyString
-        subtitle: qsTr("This password cannot be recovered. If you forget it then the wallet will have to be restored from its 25 word mnemonic seed.") + translationManager.emptyString
+        subtitle: qsTr("This password cannot be recovered. If you forget it then the wallet will have to be restored from its 25 word mnemonic seed or your hardware wallet device.") + translationManager.emptyString
```

Comment:
I think it would be better to make this context sensitive, e.g. when restoring from a hardware device it only says

> If you forget it then the wallet will have to be restored from your hardware wallet device. 

and vice versa.

Can you revert these changes? I’ll do this in a separate PR.

Reply:
You got it. Do you know if there is some existing flag that indicates when a wallet is being generated from a hardware device?

Reply:
wizardController.walletOptionsIsRecoveringFromDevice